src/backend/db/products.js

```diff
@@ -26,8 +26,8 @@ export const products = [
     description: "Orion, Astronomy Print Of Orion Constellation In The Night Sky, .",
     featuredProductDescription:
       "Orion's Belt Star Map, Astronomy Gift, Orion Nebula, Gifts For Stargazers. ",
-    price: 2000,
-    discountedPrice: 3500,
+    discountedPrice: 2000,
+    originalPrice: 3500,
```

Comment:
a very minor thing but u could've also calculated discounted price after the loading of products and added that properties to the products, that way u don't have to calculate manually here

Reply:
Ok, So that means I need to store the discount percentage in DB. I will implement that in the upcoming PR. Thanks for the review!!